packages/react-devtools-shared/src/hook.js

```diff
@@ -172,7 +173,7 @@ export function installHook(target: any): DevToolsHook | null {
     // In that case, we'll patch later (when the frontend attaches).
     //
     // Don't patch in test environments because we don't want to interfere with Jest's own console overrides.
-    if (process.env.NODE_ENV !== 'test') {
+    if (enableConsolePatching && process.env.NODE_ENV !== 'test') {
```

Comment:
Doest his actually work for you? Or does the Webpack-generated import statement just throw an error too (for the same reason as is mentioned in the inline comment:
```
        // Normally we could check "typeof patchConsole === 'function'",
        // but Webpack wraps imports with an object (e.g. _backend_console__WEBPACK_IMPORTED_MODULE_0__)
        // and the object itself will be undefined as well for the reasons mentioned above,
        // so we use try/catch instead.
```

When I tested it I see an error being thrown (and swallowed):
> ReferenceError: react_devtools_target_flags__WEBPACK_IMPORTED_MODULE_1__ is not defined